From a3188e301b5a9a1f25dbb98a510e366363348e64 Mon Sep 17 00:00:00 2001
From: Dan Dennedy <dan@dennedy.org>
Date: Sun, 19 Feb 2017 12:50:59 -0800
Subject: [PATCH] Allow Mlt::Repository to be deleted without bad side effect.

mlt_repository_close() is run by mlt_factory_close() when called either
directly, which requires all Mlt objects to be destroyed prior, or
atexit().
---
 src/mlt++/MltRepository.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/mlt++/MltRepository.cpp b/src/mlt++/MltRepository.cpp
index 47a598bc..45522e40 100644
--- a/src/mlt++/MltRepository.cpp
+++ b/src/mlt++/MltRepository.cpp
@@ -35,8 +35,6 @@ Repository::Repository( mlt_repository repository ) :
 
 Repository::~Repository( )
 {
-	if ( instance )
-		mlt_repository_close( instance );
 	instance = NULL;
 }
 
